packages/client/utils/getUniqueEdges.ts

```diff
@@ -0,0 +1,23 @@
+interface RelayEdge {
+  readonly node: Record<string, unknown> | null | undefined
```

Comment:
+1 Why `unknown` here? Doesn't this break type inference in places where it's used?

Reply:
Good point. I based my implementation off of Matt's packages/client/utils/getNonNullEdges.ts

An improvement here would be perhaps to make the interface generic, but I'll let it alone for now.